src/Fetch.js

```diff
@@ -114,7 +114,7 @@ var Fetch = {
       // either via Module.locateFile() function, or via Module.pthreadMainPrefixURL string. If neither
       // of these are passed, then the default URL 'pthread-main.js' relative to the main html file is loaded.
       if (typeof Module['locateFile'] === 'function') fetchJs = Module['locateFile'](fetchJs);
-      else if (Module['pthreadMainPrefixURL']) fetchJs = Module['pthreadMainPrefixURL'] + fetchJs;
+      else fetchJs = Module['pthreadMainPrefixURL'] + fetchJs;
```

Comment:
Here also we can get `undefined`+'someString'`, and we don't want to require the URL variable to be present.

Reply:
Same here, either `Module.locateFile` or `Module.pthreadMainPrefixURL` will be present, condition is redundant